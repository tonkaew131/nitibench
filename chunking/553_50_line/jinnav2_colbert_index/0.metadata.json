{
    "passage_offset": 0,
    "num_passages": 2610,
    "num_embeddings": 305999,
    "embedding_offset": 0
}
