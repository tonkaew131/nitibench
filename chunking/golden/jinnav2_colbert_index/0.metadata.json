{
    "passage_offset": 0,
    "num_passages": 5139,
    "num_embeddings": 603993,
    "embedding_offset": 0
}
